### PR TITLE
Add Lyft Washington, DC

### PR DESCRIPTION
**Pull request by brookemckim**

### CODE REVIEW

**Review by yocontra — APPROVED** [review]

File: systems.csv [hunk 1]
```diff
@@ -190,6 +190,7 @@ US,Las Vegas,"Las Vegas, NV",bcycle_rtcbikeshare,https://rtcbikeshare.bcycle.com
 US,Lime Bike,"Los Angeles, CA",limebike,https://www.li.me/,https://lime.bike/api/partners/v1/gbfs/los_angeles/gbfs.json
 US,Link Dayton Bike Share,"Dayton, OH",bcycle_linkdayton,https://www.linkdayton.org,https://gbfs.bcycle.com/bcycle_linkdayton/gbfs.json
 US,Louvelo,"Louisville, KY",Louvelo,https://louvelo.com/,https://lou.publicbikesystem.net/ube/gbfs/v1/
+US,Lyft,"Washington, DC",lyft_dca,https://louvelo.com/,https://s3.amazonaws.com/lyft-lastmile-production-iad/lbs/dca/gbfs.json
```
Comment (jcn) [inline systems.csv]: This system URL is not right...

**Review by jcn — CHANGES_REQUESTED** [review]

**Comment by brookemckim** [comment]
@jcn sorry about that. Fixed.

**Comment by jcn** [comment]
Sorry for being so pedantic, but I just found https://www.lyft.com/scooters/washington-dc which I think is the most appropriate URL for this entry. 

Thanks for all your help keeping this document updated!

**Comment by brookemckim** [comment]
No worries. I wasn't very clear on what that URL was supposed to point to so just kept it generic.

**Comment by jcn** [comment]
Yeah, the `systems.txt` file is meant primarily for humans, so specificity here is helpful. Thanks!

**Pull request merged**